### DISCUSSION TITLE
Prettier to sort component attributes?

### DISCUSSION THREAD

**Discussion by basememara** [mantinedev/mantine#6069] [Q&A]
I would like to enforce the sorting of the component attributes into something sane, like [Tailwind's Prettier sorting](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier). Is there something like this for Mantine?

For example, if I have the below:

```tsx
<Text my="md" size="lg" px="xl" mx="auto">
  {lorem}
</Text>
```

I'd want it to sort on save to this:

```tsx
<Text size="lg" my="md" mx="auto" px="xl">
  {lorem}
</Text>
```

Is something like this possible?

**Comment by rtivital** [comment]
It is a bad idea. https://github.com/prettier/prettier/issues/323
